zipline_poloniex/bundle.py

```diff
@@ -207,13 +219,4 @@ def day(self):
 # everywhere in run_algo._run, *DOH*!!!
 deregister_calendar('NYSE')
 register_calendar_alias('NYSE', 'POLONIEX', force=False)
-register(
-    '.test_poloniex',
-    create_bundle(
-        [Pairs.usdt_eth],
-        pd.Timestamp('2016-01-01', tz='utc'),
-        pd.Timestamp('2016-01-31', tz='utc'),
-    ),
-    calendar_name='POLONIEX',
-    minutes_per_day=24*60
-)
+# Deleted the register function as the built in zipline register function plays better with other data bundles
```

Comment:
This I don't understand. Basically `register` just registers a name for the data bundle. Why is this example `.test_poloniex` a problem here?

Reply:
I think I may have misinterpreted this. I initially thought it conflicted with ziplines default register function, but I now realize that it shouldn't. I had done these edits a while ago.